sky/server/requests/executor.py

```diff
@@ -408,9 +409,19 @@ def start(deploy: bool) -> List[multiprocessing.Process]:
         queue_names = [
             schedule_type.value for schedule_type in api_requests.ScheduleType
         ]
+        # TODO(aylei): queue manager port should be configurable.
         queue_server = multiprocessing.Process(
             target=mp_queue.start_queue_manager, args=(queue_names,))
         queue_server.start()
+
+        # Start a monitoring thread to watch queue_server process
```

Comment:
I assume the main purpose of this thread is to log the failure out in the log? If that is the case, would be nice to mention it in the comment.